docs/quickstart-aks.md

```diff
@@ -167,9 +167,11 @@ You should also ensure that the `Microsoft.Compute` and `Microsoft.Network` prov
 
 1. Create the AKS cluster!
     ```console
-    az aks create --resource-group aks-group --name osba-quickstart-cluster --generate-ssh-keys
+    az aks create --resource-group aks-group --name osba-quickstart-cluster --generate-ssh-keys --kubernetes-version 1.9.6
     ```
 
+    Note: Service Catalog may not work with Kubernetes versions less than 1.9.0. If you are attempting to use an older AKS cluster, please upgrade to at least version 1.9.1
```

Comment:
s/1.9.1/1.9.0/

Reply:
AKS does not have 1.9.0, so 1.9.1 is the minimum they can upgrade to.

Reply:
Can you amend this like:

"... please upgrade to at least version 1.9.1-- which is the oldest compatible version offered by AKS."